### PR TITLE
Command to message conversion

### PR DESCRIPTION
**Pull request by DejaToris**
The base commands have been converted 👾

### CODE REVIEW

File: src/valet_parking_slack_bot/logic.py [hunk 1]
```diff
@@ -1,26 +1,30 @@
...
         self.parking_spot_repo = parking_spot_repo
 
+    def spots(self):
+        number_of_spots = len(self.parking_spot_repo.retrieve_available_spots())
+        return number_of_spots
 
-    def try_reserve_spot(self, username):
+    def try_reserve_spot(self, user_id):
         available_spots = self.parking_spot_repo.retrieve_available_spots()
         if len(available_spots) == 0:
```
Comment (TheCoreMan) [inline src/valet_parking_slack_bot/logic.py]:
```suggestion
        available_spots: List[Any] = self.parking_spot_repo.retrieve_available_spots()
        if not available_spots:
```

File: src/valet_parking_slack_bot/logic.py [hunk 2]
```diff
@@ -1,26 +1,30 @@
...
-            return "User had no assigned parking"
-        elif len(user_reserved_spots) == 1:
+    def release_by_user_id(self, user_id):
+        user_reserved_spots = self.parking_spot_repo.retrieve_spots_by_user(user_id)
+        if len(user_reserved_spots) == 1:
             self.parking_spot_repo.release(user_reserved_spots[0])
-            return f"Parking spot {user_reserved_spots[0]} has been released successfully"
-        else:
-            return f"You have several reserved spots: {user_reserved_spots}. Which one to release?"
+            logger.info(f'Attempting to release spot {user_reserved_spots[0]}')
```
Comment (TheCoreMan) [inline src/valet_parking_slack_bot/logic.py:26]: Shouldn't you call "release" here?

**Review by TheCoreMan — COMMENTED** [review]
Looks pretty good! :)

File: src/valet_parking_slack_bot/logic.py (hunk 2 above)
Comment (DejaToris) [reply src/valet_parking_slack_bot/logic.py:26]: I do 🤔

**Pull request merged**